Addons/testAddons.cpp

```diff
@@ -53,29 +58,38 @@ int main(int argc, char const **argv){
         args.pop_front();
     }
 
-    bool pass = testConstructSurrogate(verbose);
-    cout << std::setw(40) << "Automated construction" << std::setw(10) << ((pass) ? "Pass" : "FAIL") << endl;
-    pass_all = pass_all && pass;
-
-    #if defined(Tasmanian_ENABLE_BLAS) || defined(Tasmanian_ENABLE_GPU)
-    pass = true;
-    pass = testLoadUnstructuredL2(verbose, gpuid);
-    cout << std::setw(40) << "Unstructured construction" << std::setw(10) << ((pass) ? "Pass" : "FAIL") << endl;
-    pass_all = pass_all && pass;
-    #else
-    cout << std::setw(40) << "Unstructured construction" << std::setw(10) << "skipping" << endl;
-    gpuid *= 2; // no op to register the use of gpuid
-    #endif
-
-    cout << "\n";
-    if (pass){
+    if (debug) {
```

Comment:
No need to shift the alignment here, just put
```
if (debug){
    debugTest();
    return 0;
}
```

Reply:
Changed.